### DISCUSSION TITLE
linux-experience-part2/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#39] [Announcements]
# linux-experience-part2/

تو پست قبل راجع به لینوکس و بخشی از مسیری که باهاش تجربه کردم، صحبت کردم.
گفتم که از توزیع‌هایی مثل Kubuntu که از KDE + Ubuntu تشکیل شدند شروع کردم و بعداً تو ادامه مسیر سراغ توزیع های مبتنی بر آرچ رفتم.
برام جالب بود که بعضی‌ها که آرچ داشتند، انگار هویت‌شون رو با سیستم‌عامل‌شون تعریف می‌کردند. اصطلاحی هم هست که بعضی‌ها میگن:
I use arch btw (by the way).

https://aprd.ir/linux-experience-part2/

**Comment by mreleeam** [comment]
سلام مهندس جان برای VPN من OpenVPN میزنم (ProtonVPN) که رایگانه و یه HTTP Proxy از WebShare که اینم رایگان هست بهش اضافه میکنم و کل آیپی های داخلی رو هم روت استاتیک میزنم روی Default Gateway و ۲ سالی میشه خیلی خوب کار میکنه روی اوبونتو.البته از V2Ray و Shadowsocks و هر چیز دیگه ای HTTP Proxy بده هم میشه استفاده کرد.
راستش الان که میبینم حیف این ۴ سال که روی شبکه و سیستم وقت گذاشتم روی ML وقت میذاشتم الان آینده بهتری داشتم شایدم خیلی بد بینم با اینکه ۱۸ سالمه ولی خوب حس میکنم خیلی دیر شده برای من که AI رو شروع کنم چون هیچ ایده ای ندارم که قالب عموم مردم چقدر زودتر از من AI رو شروع کردن و اصلا نمیدونم چقدر زمان میبره تا به یک سطح متوسط از AI برسم

**Reply by pourmand1376** [reply]
سلام ایلیا. وقتت بخیر. 

شما به خوبی به نکته‌ای که توی لینوکس مدنظرم بود اشاره کردید. جامعه لینوکس کارها همچین کارهایی براشون عادیه و اصلاً یه جورایی انتظار می‌ره که این کارها رو بکنند. 

ولی من از یک جایی به بعد خسته شدم و راستش دیگه حوصله تجربه اون فضا رو ندارم. دوست دارم یک چیزی داشته باشم که به قول خارجی‌ها «It just works». 

به نظرم سر و کله زدن با این چیزها اگر چه خوبه و کلی یادگیری می‌تونه داخلش داشته باشه، ولی چون محدودیت منابع داریم، ترجیح میدم روی یادگیری چیزهای دیگه ای وقت بگذارم. 

راجع به هوش مصنوعی هم آنقدر دنیای بزرگی هست که هر کسی یک چیز خیلی خیلی کوچیکش رو شروع کرده و داره کار می‌کنه. 

مخصوصاً بحث LLMها که الان مد روز شده رو اگر نگاه کنی، هنوز شرکت‌های بزرگ هم توش موندن و دارن سعی و خطا می‌کنند. اینه که اگر شما هم بیای به جمع سعی و خطا کنندگان اضافه می‌شد😉

البته یک سری مفاهیم پایه هست که بهتره بدونی و اصطلاحاً خاک حوزه رو بخوری ولی اگر بتونی وقت بگذاری، چیز عجیب غریبی نیست. اصلاً هم این حس بهت دست نده که دیر شده.

اتفاقاً اگر آدم بتونه از ۱۸ سالگی بفهمه که در آینده می‌خواد چیکار کنه، خیلی زود فهمیده. اگر چه اکثر آدم‌هایی که من می‌شناسم (از جمله خودم)، ۴ ۵ سالی همین‌طور تو حوزه‌های مختلف جسته گریخته کار کردند تا حدوداً متوجه بشن چی دوست دارند. 

امیدوارم که تو پیدا کردن مسیر شغلی‌ برای خودت، موفق باشی🙂 و بتونی شغل مورد علاقه‌ات (نه لزوماً اونی که ترند شده) رو پیدا کنی و توش فعالیت کنی.

**Comment by alidalvand** [comment]
وقتی صحبت از سیستم‌عامل‌ها می‌شود، اغلب لینوکس را به عنوان انتخابی برای متخصصان فناوری می‌شناسیم. اما آیا تا به حال فکر کرده‌ایم که کاربران عادی، مثل کارمندان ادارات و افراد معمولی، چه انتظاری از یک سیستم‌عامل دارند؟ بیشتر کاربران به دنبال یک سیستم‌عاملی هستند که کارهای روزمره آن‌ها را ساده کند. آن‌ها می‌خواهند به راحتی فیلم تماشا کنند، موسیقی گوش دهند، در اینترنت گشت و گذار کنند و از نرم‌افزارهای آشنایی مثل ورد و اکسل استفاده کنند. ویندوز به دلیل سادگی و گستردگی نرم‌افزارهایش، تا حد زیادی این نیازها را برطرف کرده است. برای اینکه لینوکس بتواند به یک جایگزین مناسب برای ویندوز تبدیل شود، نیاز به آموزش‌های ساده و جامع و همچنین یک محیط کاربری جذاب و شهودی دارد. اینجوری کاربران عادی هم می‌توانند بدون دردسر از مزایای لینوکس بهره‌مند شوند. عادت کردن به یک سیستم‌عامل خاص، به خصوص برای نسلی که با ویندوز و بازی های آن بزرگ شده، کار ساده‌ای نیست. بازی‌ها و نرم‌افزارهای محبوب ویندوز، نقش مهمی در این وابستگی دارند.
